website/developer-docs/blueprints/v1/structure.md

```diff
@@ -1,10 +1,34 @@
+import Collapse from "@site/src/components/Collapse";
+
 # File structure
 
 Blueprints are YAML files, which can use some additional tags to ease blueprint creation.
 
 ## Schema
 
-The blueprint schema is available under `https://goauthentik.io/blueprints/schema.json`. It is also possible to target a specific version's blueprint schema by using `https://version-2023-4.goauthentik.io/blueprints/schema.json`.
+The blueprint schema is available under
+[`https://goauthentik.io/blueprints/schema.json`](https://goauthentik.io/blueprints/schema.json). It
+is also possible to target a specific version's blueprint schema by using
+[`https://version-2023-4.goauthentik.io/blueprints/schema.json`](https://version-2023-4.goauthentik.io/blueprints/schema.json).
+
+<Collapse title="Available Older Blueprint Schemas">
+- [Version 2022-8](https://version-2022-8.goauthentik.io/blueprints/schema.json)
```

Comment:
We really need to have some sort of automation to generate a list of versions instead of hardcoding this, I can already for see this getting outdated (and also maybe order in the reverse of this starting with current and getting older?)